### DISCUSSION TITLE
Probability of genotypes with AlphaImpute2

### DISCUSSION THREAD

**Discussion by Leo4Luffy** [AlphaGenes/AlphaImpute2#24] [Q&A] [answered]
Good morning, I hope you are very well. I just wanted to ask a little question. I have performed the imputation of genomic data using AlphaImpute2. I have used the three proposals of this program: imputation based on the pedigree, imputation based on the population and imputation using both sources of information. From this, I have obtained the genotypes in phase and the imputed genotypes. I wanted to ask you, and excuse me, if AlphaImpute2 also allows you to obtain the probability of genotypes as AlphaImpute does? I would appreciate any information you can give me. A happy day.

**Comment by gregorgorjanc** [comment, marked as answer]
Hi @Leo4Luffy,

The model and algorithms behind AlphaImpute2 are probabilistic https://www.biorxiv.org/content/10.1101/2020.09.16.299677v1, but (to my understanding) output are called alleles and genotypes. You have two options:

1) Play with call thresholds

2) Look into calling code and output the underlying probabilities

**Comment by gregorgorjanc** [comment]
@Leo4Luffy I have opened an issue for this at https://github.com/AlphaGenes/AlphaImpute2/issues/25
